biggus/__init__.py

```diff
@@ -2560,6 +2726,30 @@ def sub(a, b):
     return _Elementwise(a, b, np.subtract, np.ma.subtract)
 
 
+def multiply(a, b):
+    """
+    Return the elementwise evaluation of `a * b` as another Array.
+
+    """
+    return _Elementwise(a, b, np.multiply, np.ma.multiply)
+
+
+def divide(a, b):
+    """
+    Return the elementwise evaluation of `a / b` as another Array.
+
+    """
+    return _Elementwise(a, b, np.divide, np.ma.divide)
```

Comment:
What a bomb-shell :bomb::

```
>>> np.divide(2, 4)
0
```

This suggests that my use of ``__truediv__`` is erroneous. :cry: